pypika/clickhouse/nullable_arg.py

```diff
@@ -0,0 +1,28 @@
+from pypika import Field
+from pypika.terms import Function
+
+
+class IfNull(Function):
+    def __init__(self, field, alt, alias: str = None, schema: str = None):
```

Comment:
if you change this to 
```
 def __init__(self, *args, **kwargs):
     super().__init__('NULLIF', name, alias=alias)
```

then you wouldn't need to write a custom implementation for `get_sql`

Reply:
if you want to enforce the args, you could do it like
```
 def __init__(self, term, alt, **kwargs):
    super().__init__('NULLIF', term, alt, **kwargs)
```